web/src/main/java/org/openmrs/web/filter/initialization/InitializationFilter.java

```diff
@@ -1133,6 +1153,12 @@ private void importTestDataSet(InputStream in, String connectionUrl, String conn
 		}
 	}
 	
+	private boolean checkDatabase(String database) {
```

Comment:
What does checkDatabase mean?

Reply:
Given a string as input , checkDatabase checks whteher the database is required or supported by OpenMRS platform.

Reply:
Shouldn't the method name contain that information such that one does not need this comment to understand what the method does?